docs/decisions/0002-push-notifications.rst

```diff
@@ -0,0 +1,56 @@
+Implement Push Notification Channel
+==================================================
+
+Status
+------
+Accepted
+
+Context
+------
+The goal is to enhance the existing notification framework with push notifications.
+Flexibility and seamless integration with the existing framework are priorities
+for this new notification channel. Currently, edx-ace supports various email channels
+like django and Sailthru, but lacks support for mobile push notifications. This feature will
+enable real-time communication with users through their mobile devices, enhancing user
+engagement and ensuring timely delivery of important information.
+
+Decision
+------
+Introduce a new push notification system.
+
+This will involve:
```

Comment:
Please add more description about what actions we should do to make push notifications delivery channel work for people. Including API endpoints and Push notifications rendered templates that should be coming from outside of ACE framework.

Reply:
Done